test-cases/features/repair-based-operations/longevity-100gb-4h-ReplaceNode.yaml

```diff
@@ -1,4 +1,4 @@
-test_duration: 360
+test_duration: 120
```

Comment:
Why would you want to do  it?
It won't finish to run.

Reply:
What do you mean by "won't finish to run". Did you mean it would not finish any replace operations?

I want to reduce the time because I only care about the first 2 or 3 replace operations. It is more than enough to get what I want know: how the system is going during replace operations.

I do not need to run the replace operations 12 times to know the latency / throughput of the system.

Reply:
This is not a latency/throughput test.
It's a longevity (or sanity) of 4h.

We may change it and call it as something else, but this is not the purpose of your tests.
AFAIU, the purpose is to start with the basic ones like these tests, then progress to the more complex  and longer ones till we are sure we can enable it on master.

Performance is lower priority.

Reply:
It is 360 minutes / 60 = 6 hours, if I do not calculate it wrongly. 

We have run the full 6 hours test multiple times. All the replace operations have passed in those runs. I want to use this test to find more issues, not just the operations passes or not. 

Performance does matter. If we have huge latency spikes or big OPS drop we can not enable it by default anyway. 

Roy, you can test whatever you think it is necessary from QA's point of view. But please let me test what I needed most at the moment. I want to find whatever issues we could find.

Reply:
The whole idea is that after  test passes, to leave it like this for  regression testing.
Eventually we want to merge this branch back to master and use these jobs to test RBNO before we release.
Thus, it's not a good way to change these tests  after  we "certified" them.
It's fine to go and add another test to do whatever you want.
For performance, I told you that @amoskong has a  better test to be used, but you can send whatever you want, just  as a separate test.

Reply:
How do I test whatever I want to test if you do not merge my patches for rbno. Can you change  or add a new Jeknins test to point to my own branch?

Reply:
@roydahan @slivne ping

Reply:
For playing with the tests you can use something like: https://jenkins.scylladb.com/view/master/job/scylla-master/job/longevity/job/byo-longevity-test/

The final tests that needs to pass the "bar" in order to be merged to master, are the tests that I defined originally.